misc/formatter/modules/formatter-core/src/test/resources/declarations/class-definition/source/class_definition_declaration_18.bal

```diff
@@ -0,0 +1,15 @@
+class Student {
+    string name;
+    int age;
+
+    public function init(string name, int age) {
+        self.name = name;
+        self.age = age;
+    }
+    public function getName() returns string {
+        return self.name;
+    }
+    public function getAge() returns int {
+        return self.age;
+    }
+}
```

Comment:
Do we have a test case to check what happens if there are more than one blank lines? (if not, probably we can use this same test resource to verify the output in that scenario as well)